src/sgxlkl.edl

```diff
@@ -31,6 +31,11 @@ enclave {
         // Enclave call for initializing ethreads to enter enclave
         public void sgxlkl_ethread_init(void);
 
+        // Enclave call to dump stack traces for all lthreads (DEBUG only)
+        // TODO: This should only be included for a DEBUG build, but EDL doesn't seem to
```

Comment:
It does have include.  We could have a separate top-level EDL file for debug and non-debug builds and include the non-debug one in the debug build.

Reply:
Ah, ok. I suggest that this should be a separate PR though, as it's related to #144 and #171.

Reply:
No objections.